dev-tools/ecs-migration.yml

```diff
@@ -10,3 +10,20 @@
 #   alias: true-if-alias-is-required-in-6x
 #   # Copy to is useful for fields where multiple fields map to the same ECS field
 #   copy_to: true-if-field-should-be-copied-to-target-in-6x
+#
+
+
+- from: beat.name
+  to: agent.name
```

Comment:
This is not aligned with the changelog. Shouldn't "beat.name" become "beat.type"?

Reply:
yes, `agent.type`